index/src/executors/dex_executors/acala_swap.rs

```diff
@@ -0,0 +1,228 @@
+use pink_subrpc::{create_transaction, send_transaction, ExtraParam};
+
+use crate::prelude::DexExecutor;
+use crate::traits::common::Error;
+use alloc::{
+    string::{String, ToString},
+    vec,
+    vec::Vec,
+};
+use scale::Decode;
+use scale::{Compact, Encode};
+
+#[derive(Clone)]
+pub struct AcalaDexExecutor {
+    rpc: String,
+}
+
+#[allow(dead_code)]
+impl AcalaDexExecutor {
+    pub fn new(rpc: &str) -> Self
+    where
+        Self: Sized,
+    {
+        Self {
+            rpc: rpc.to_string(),
+        }
+    }
+}
+
+impl AcalaDexExecutor {
+    #[allow(dead_code)]
+    fn aggregated_swap(
+        &self,
+        signer: [u8; 32],
+        path: Vec<u8>,
+        spend: u128,
+        _recipient: Vec<u8>,
+    ) -> core::result::Result<(), Error> {
+        let amount_out = Compact(1_u8);
```

Comment:
Still the same question as above, in my understanding, unless we call `swapWithExactTarget`, we can not force the out amount.